File: 1013-fibonacci-number/fibonacci-number.cpp
class Solution {
public:
    int findFibonacci(int n)
    {
        if(n<=1)
        {
            return n;
        }
        return findFibonacci(n-1)+findFibonacci(n-2);
    }
    int fib(int n) 
    {
       return findFibonacci(n); 
    }
};

File: 1013-fibonacci-number/fibonacci-number.java
class Solution {
    public int findFib(int n)
    {
        if(n<=1)
        {
            return n;
        }
        int first=findFib(n-1);
        int last=findFib(n-2);

        return first+last;
    }
    public int fib(int n) 
    {
        return findFib(n);
    }
}